blocks/init/src/Blocks/components/modal/modal-style.scss

```diff
@@ -0,0 +1,88 @@
+.modal {
+	$this: &;
+
+	--modal-scoped-icon-size: 1.5rem;
+
+	display: none;
+
+	&.is-open {
+		display: block;
+	}
+
+	&__overlay {
+		position: fixed;
+		top: 0;
+		left: 0;
+		right: 0;
+		bottom: 0;
+		z-index: 999;
+		background-color: rgba(0, 0, 0, 0.5);
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		will-change: transform;
+	}
+
+	&__dialog {
+		min-width: 360px;
+		max-width: calc(100% - 1.25rem);
+		max-height: 90%;
+		overflow: auto;
+		position: relative;
+		padding: 1.25rem;
+		will-change: transform;
```

Comment:
No need to add that, that's a "last measure" performance optimization (https://developer.mozilla.org/en-US/docs/Web/CSS/will-change) Plus you shouldn't really have any lag issues with animating transforms (`opacity` and `transform` are generally the only things you should animate if possible).